napi-inl.h

```diff
@@ -3705,6 +3705,28 @@ inline const napi_node_version* VersionManagement::GetNodeVersion(Env env) {
   return result;
 }
 
+#if (NAPI_VERSION > 2)
+////////////////////////////////////////////////////////////////////////////////
+// Cleanup class
+////////////////////////////////////////////////////////////////////////////////
+
+inline CleanupHook::CleanupHook(Env env) : _env(env) {
+  napi_status status = napi_add_env_cleanup_hook(_env, NapiHook, static_cast<void*>(this));
```

Comment:
According to the document[1], `napi_add_env_cleanup_hook` can safely be specified multiple times with different `arg` values. But I guess your code doesn't allow it, right? Shouldn't we consider a situation that the method is called multiple times?